### PR TITLE
Gay Boykissers Round 2

### PR DESCRIPTION
**Pull request by BackeTako**
## About the PR

Added a male gay pride scarf

## Why / Balance

Boykissers rejoice, yet again.
space-wizards/space-station-14/pull/32584

## Media

gay_scarf

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None that #216 wouldn't have already caused.

**Changelog**

Added a male Gay scarf with the other pride ones in the winterdrobe.

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@
     ClothingNeckScarfStripedAce: 3
     ClothingNeckScarfStripedAro: 3
     ClothingNeckScarfStripedBiSexual: 3
+    ClothingNeckScarfStripedGay: 3
```
Comment (LankLTE) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml:28]: Please put a comment indicating this is a CD change

**Review by LankLTE — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml (hunk 1 above)
Comment (BackeTako) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml:28]: that is marked as a CD change

Comment (dffdff2423) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml:28]: While you are looking at this, could you add a comment where the CD content ends?

**Review by dffdff2423 — APPROVED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml (hunk 1 above)
Comment (BackeTako) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml:28]: Done it

**Review by LankLTE — APPROVED** [review]

**Pull request merged**